tools/editor/multi_node_edit.cpp

```diff
@@ -52,8 +52,16 @@ bool MultiNodeEdit::_set(const StringName& p_name, const Variant& p_value){
 		Node*n=es->get_node(E->get());
 		if (!n)
 			continue;
+		else if (p_value.get_type() == Variant::NODE_PATH) {
+			Node *tonode = n->get_node(p_value);
+			if (tonode) {
+				NodePath p_path = n->get_path_to(tonode);
+				ur->add_do_property(n,name,p_path);
+			}
```

Comment:
yes, that makes sense, just did that. The reason why I had the if was that I copied the code from `tools/editor/property_editor.cpp` which uses an if. But who wrote that probably forgot about the error checking inside `get_node`, `get_path_to`, etc